backend/src/StdLibDarkInternal/Helpers/ProgramTypesToDarkTypes.fs

```diff
@@ -827,8 +828,12 @@ module Expr =
       PT.EVariable(uint64 id, varName)
 
     // control flow
-    | DEnum(_, _, "EIf", [ DInt id; cond; ifTrue; ifFalse ]) ->
-      PT.EIf(uint64 id, fromDT cond, fromDT ifTrue, fromDT ifFalse)
+    | DEnum(_, _, "EIf", [ DInt id; cond; thenExpr; elseExpr ]) ->
```

Comment:
I think this is the cause of the errors in CI - we need to read from options in `fromDT` like we write to them in `toDT`